common/src/blinker.rs

```diff
@@ -0,0 +1,18 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Blinker {
```

Comment:
I don't have strong opinions on this, but there are some options (not mutually exclusive) for what to do with this file:
- Most components live in `proto.rs`. This _can_ live there, too, but it doesn't have to. I have a PR that makes `Material` a component, so this isn't a hard and fast rule.
- Since all fields are public, having a constructor is optional, as it can be constructed manually in `server::sim` instead.

Reply:
Noted, those are good patterns.